backend/app/views/spree/admin/customer_returns/new.html.erb

```diff
@@ -33,7 +33,7 @@
         </fieldset>
 
         <%= f.field_container :stock_location do %>
-          <%= f.label :stock_location, Spree.t(:stock_location) %>
+          <%= f.label :stock_location_id %>
```

Comment:
Its the name of the other model. Its referencing what StockLocation it belongs_to, so referencing what a StockLocation is by the way of that table is perfectly reasonable